libcontainer/cgroups/fs2/memory.go

```diff
@@ -99,9 +103,15 @@ func statMemory(dirPath string, stats *cgroups.Stats) error {
 	if err != nil {
 		return err
 	}
+	// As cgroup v1 reports SwapUsage values as mem+swap combined,
```

Comment:
Should we expose a separate field that returns just the swap for consumers? (Not a blocker for this PR)

Reply:
Alas, the field that holds swap+mem usage is inconveniently called "Swap".

So, we could add something like "SwapOnly" (for both v1 and v2) or "SwapV2" (for v2 only), or something entirely different -- I am very open to suggestions.